src/components/sortable-list/index.tsx

```
@@ -0,0 +1,219 @@
+    };
+
+    const itemIds = items.map( keyExtractor );
+    const containerNamespace = kebabCase( namespace );
+
+    const getLayoutClasses = () => {
+        switch ( strategy ) {
+            case 'horizontal':
+                return 'flex flex-row items-center';
+            case 'grid':
+                return `grid grid-cols-1 sm:grid-cols-2 md:grid-cols-${ gridColumns } lg:grid-cols-${ gridColumns }`;
+            case 'vertical':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)